server/services/websockets.js

```diff
@@ -79,6 +79,11 @@ export default class Websockets {
             ],
           });
       }
+      case "documents.permanent_delete": {
+        return socketio.to(`team-${event.teamId}`).emit(event.name, {
```

Comment:
```suggestion
        return socketio.to(`collection-${document.collectionId}`).emit(event.name, {
```

I believe this should be sent down the collection channel otherwise the event is propagated to users that don't have permission to know about the document. (It's only an id down a websocket, but same principles apply)